tstest/clock.go

```diff
@@ -303,6 +303,11 @@ func (c *Clock) AfterFunc(d time.Duration, f func()) tstime.TimerController {
 	return t
 }
 
+// Since subtracts specified duration from Now().
```

Comment:
there is no duration specified here.

How about we copy the docs from the standard library, which says:

> Since returns the time elapsed since t. It is shorthand for time.Now().Sub(t).